app/assets/javascripts/discourse/app/components/topic-timeline/container.hbs

```diff
@@ -32,6 +32,7 @@
 
 {{#if (and (not @fullscreen) @currentUser)}}
   <div class="timeline-controls">
+    <PluginOutlet @name="timeline-controls-before" @tagName="span" @connectorTagName="span" @args={{hash model=@model}} />
```

Comment:
Let's drop the `tagName` and `connectorTagName`. The default now is for tagless outlets, which give the most control to plugins/themes. (and are the most glimmer-like)